src/npc.py

```diff
@@ -0,0 +1,85 @@
+""" NPC Module
+    Defines NPC for special rooms """
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+from src.character import Character
+
+class NPC(Character):
+    # NPC superclass creates an NPC for special rooms
+    def __init__(self, name, hp):
```

Comment:
I think it would make sense for the hp parameter to be defaulted to some value since a NPC has no need for hp anyways.